pkg/hub/managedclusterset/default_managedclusterset_controller.go

```diff
@@ -19,7 +19,11 @@ import (
 	clusterv1beta1 "open-cluster-management.io/api/cluster/v1beta1"
 )
 
-const defaultManagedClusterSetName = "default"
+const (
+	defaultClusterSetAnnotation    = "open-cluster-management.io/autoupdate"
+	defaultClusterSetDisabledValue = "false"
```

Comment:
Just as @qiujian16 Sailed, we probably have other bootstrapped clusterset in the future. And this annotation may be used for other sets.
So I suggest change the var name as: `autoUpdateAnnotation` and there is no need to define the value. just use `false` in code logic.

Reply:
done https://github.com/open-cluster-management-io/registration/pull/210/files#diff-83468281441062b8e876bcef48f61044f897afce081d9790253027dacd34fc78R23